lib/stat_tracker.rb

```diff
@@ -148,9 +148,10 @@ def game_result?(result, team_id, game_id)
     game.result == result
   end
 
-  def opponents_and_num_result_for_team(team_id, game_result)
+  def opponents_and_opponents_num_result_for_team(team_id, game_result)
     opponent_by_game_id_for_team(team_id).reduce(Hash.new(0)) do |result, game_id_oppo_ary|
-      result[game_id_oppo_ary[1]] += 1 if game_result?(game_result, team_id, game_id_oppo_ary[0])
+      result[game_id_oppo_ary[1]] += 0 
+      result[game_id_oppo_ary[1]] += 1 if game_result?(game_result, game_id_oppo_ary[1], game_id_oppo_ary[0])
```

Comment:
Nice job here. I'm guessing the order here let you avoid an if/else statement right?

Reply:
yup exactly!